### PR TITLE
feat: add ffmpeg-python recipe

### PR DESCRIPTION
**Pull request by Faylixe**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Faylixe** [comment]
@conda-forge/staged-recipes could you please review and merge :)

File: recipes/ffmpeg-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - ffmpeg
+    - future
```
Comment (scopatz) [inline recipes/ffmpeg-python/meta.yaml]: this seems to be missing a lot of dependencies

Comment (Faylixe) [reply recipes/ffmpeg-python/meta.yaml]: It is simply a wrapper based on FFmpeg process. So there is no dependency (see repository setup.py). Only pytest-runner 
as setup dependency and future as running dep

File: recipes/ffmpeg-python/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - ffmpeg
+
+about:
+  home: https://github.com/kkroening/ffmpeg-python
+  license: Apache 2.0
```
Comment (marcelotrevisani) [inline recipes/ffmpeg-python/meta.yaml]:
```suggestion
  license: Apache-2.0
```

**Comment by Faylixe** [comment]
Is everything ok now ?

**Pull request merged**